### ISSUE TITLE
new relation-type 

### ISSUE THREAD

**Issue by BaptisteCecconi** [ivoa-std/VOResource#10]
Obscore and EPNcore services are presenting metadata about data products (images, spectra, cubes, etc). Those data products may have a DOI (or be part of a collection with a DOI) provided by a data repository. In this case, we should be able to relate the service to those DOI. It is possible to expose relationships, but in such a case, none of the current relationship-type values are not fully satisfactory. In this case, `isMetadataFor` would be applicable (it is part of the Datacite relationtype keywords).

**Comment by msdemlei** [comment]
On Mon, May 20, 2024 at 05:26:13PM -0700, Baptiste Cecconi wrote:
> not fully satisfactory. In this case, `isMetadataFor` would be
> applicable (it is part of the Datacite relationtype keywords).

Are you suggesting that dataset DOIs should be managed in the
Registry?  Wouldn't these be seriously many?  If I understand right
what you are asking for here, I'd say this is exactly what BibVO was
written for.  If I misunderstand what you want: Do you have a
concrete example?

**Comment by BaptisteCecconi** [comment]
I don't think BibVO is meant to do this (or should I read it again): if I publish a biblink associating a dataset DOI to an ivo-id of an EPNcore table, NASA/ADS won't use it, since they don't index datasets in their database. They only care about paper DOIs/bibcodes.

Here is a use case: The resource `ivo://padc.obspm.maser/cassini_rpws/epn/epn_core` is metadata for datasets https://doi.org/10.25935/XS9J-ND90, https://doi.org/10.25935/F8NS-0911, https://doi.org/10.25935/F8NS-0911, https://doi.org/10.25935/5JFX-DH49, https://doi.org/10.25935/9ZAB-FP47 and https://doi.org/10.25935/ZKXB-6C84. All those datasets are in the same table, since they are different processing levels of the same data.

I can also declare the reverse relation in Datacite, with `hasMetadata` relation-type, but the ivo-id is not so much usable outside the VORegistry, so I would prefer to have it on both sides.

**Comment by BaptisteCecconi** [comment]
Here is another example, presented in RDF/Turtle: 

```
@prefix vor: <http://www.ivoa.net/xml/VOResource/v1.0#> .
@prefix vods: <http://www.ivoa.net/xml/VODataService/v1.0#> .
@prefix vorel: <http://www.ivoa.net/rdf/voresource/relationship_type#>

<ivo://padc.obspm.maser/tfcat/q/epn_core> a vods:CatalogResource ;
    vorel:isMetadataFor <https://doi.org/10.25935/1fqm-fs07>, 
        <https://doi.org/10.25935/nhb2-wy29>, 
        <https://doi.org/10.25935/R11G-6J63>, 
        <https://doi.org/10.25935/ZMMZ-CA38>, 
        <https://doi.org/10.25935/T033-QS72>, 
        <https://doi.org/10.25935/8MAY-4320> .

<ivo://padc.obspm.maser/tfcat/q/leblanc_nda_jupiter_emi> a vods:CatalogResource ;
    vorel:isSupplementTo <https://doi.org/10.25935/1fqm-fs07> .

<ivo://padc.obspm.maser/tfcat/q/louis_juno_waves> a vods:CatalogResource ;
    vorel:isSupplementTo <https://doi.org/10.25935/nhb2-wy29> .

<ivo://padc.obspm.maser/tfcat/q/taubenschuss_frp_saturn> a vods:CatalogResource ;
    vorel:isSupplementTo <https://doi.org/10.25935/R11G-6J63> .

<ivo://padc.obspm.maser/tfcat/q/wu_skr_harmonic> a vods:CatalogResource ;
    vorel:isSupplementTo <https://doi.org/10.25935/T033-QS72> .

<ivo://padc.obspm.maser/tfcat/q/wu_nkom_saturn> a vods:CatalogResource ;
    vorel:isSupplementTo <https://doi.org/10.25935/ZMMZ-CA38> . 

<ivo://padc.obspm.maser/tfcat/q/wu_anomalous_nsmr> a vods:CatalogResource ;
    vorel:isSupplementTo <https://doi.org/10.25935/8MAY-4320> .
```

**Comment by BaptisteCecconi** [comment]
yes, that could be managed through a VEP, indeed, if I'm able to put DOIs in the IVOA registry records of my CatalogueResource, as shown in the examples. I mean something like:

```
    <relationship>
      <relationshipType>IsMetadataFor</relationshipType>
      <relatedResource uri="https://doi.org/10.25935/XS9J-ND90">Cassini/RPWS/HFR LESIA/Kronos N2 Data Collection V1.0</relatedResource>
    </relationship>
```

**Comment by msdemlei** [comment]
On Mon, Oct 14, 2024 at 08:21:05AM -0700, Baptiste Cecconi wrote:
> yes, that could be managed through a VEP, indeed, if I'm able to
> put DOIs in the IVOA registry records of my CatalogueResource, as
> shown in the examples.

This is the intent of this change, except we want the non-URI form of
the DOI, i.e.:

>     <relationship>
>       <relationshipType>IsMetadataFor</relationshipType>
>       <relatedResource uri="doi:10.25935/XS9J-ND90">Cassini/RPWS/HFR LESIA/Kronos N2 Data Collection V1.0</relatedResource>
>     </relationship>

Whether or not this is a useful thing to do (who will see these
relationships when?) is a different matter.  I still suspect that
what you have here is more a matter for TableReg
<https://ivoa.net/documents/Notes/TableReg/>, where people then will
find it in the ADS.  But I will freely admit that I have not tried
too hard to understand where this is all going.
